lively.freezer/src/util/load-template.html

```diff
@@ -9,7 +9,7 @@
   <meta http-equiv="refresh" content="0;url=noscript.html">
   </noscript>
   <title>__TITLE_TAG__</title>
-  <link type="text/css" rel="stylesheet" id="lively-font-bundle" href="assets/fonts/fonts.css">
+  <link type="text/css" rel="stylesheet" id="lively-font-bundle" href="assets/fonts/fonts.css" onload="document.LIVELY_FONTS_LOADED = true">
```

Comment:
@merryman thinking about it we could also use the lively object here probably? I do not really care as this is something that should not be used anywhere else anyways 🤷

Reply:
We can do that, I would just prefer moving the declaration of the `lively = {}` global in front of the link tag, to make sure we have no race condition, although I deem that unlikely tbh.